apps/notifications/src/panel/templates/index.jsx

```diff
@@ -421,7 +420,7 @@ class Layout extends React.Component {
 		const notes = this.filterController.getFilteredNotes( allNotes );
 		if (
 			this.state.selectedNote &&
-			find( notes, matchesProperty( 'id', this.state.selectedNoteId ) ) === null
+			notes.find( ( n ) => n.id === this.state.selectedNoteId ) === undefined
```

Comment:
Is this supposed to be `null`  or `undefined`? Should we just do `== null` to cover both cases?

Reply:
I'm actually not sure the existing code was ever guaranteed to work? The docs for `_.find` say that it "Returns the matched element, else `undefined`." This would mean the old expression would never evaluate to true -- I _think_ -- since it has to find an element in the list that's null, but which also has an ID that matches the selected note ID.